gel-cli-instance/src/local/localbackup.rs

```diff
@@ -367,30 +702,41 @@ impl<P: ProcessRunner> PgBackupCommands<P> {
         Ok(())
     }
 
-    // pub async fn pg_restore(&self, callback: ProgressCallback) -> Result<(), ProcessError> {
-    //     unimplemented!()
-    // }
+    pub async fn pg_combinebackup(
+        &self,
+        target_dir: impl AsRef<Path>,
+        paths: &[impl AsRef<Path>],
+        callback: ProgressCallback,
+    ) -> Result<(), anyhow::Error> {
+        let pg_combinebackup = self.find_executable("pg_combinebackup")?;
+        let mut cmd = Command::new(pg_combinebackup);
+        cmd.arg("--output").arg(target_dir.as_ref());
```

Comment:
Consider using `--clone` (works on MacOS and XFS on Linux).

Reply:
I originally tried adding per-platform flags here but `pg_combinebackup` doesn't do fallback and error recovery was challenging. We might need to do some sort of capability sniffing. I'm not totally sure why they don't do that internally in the tool.